terraform/ec2/ec2.tf

```diff
@@ -0,0 +1,97 @@
+provider "aws" {
```

Comment:
Looks like we'll save a lot by deleting this folder! https://github.com/aws-observability/aws-otel-collector-test-framework/tree/master/integ-test/src/main/java/com/amazon/aocagent/services

Reply:
yeah, it does!! i will provide ecs and eks soon